solutions/fully-configurable/variables.tf

```diff
@@ -89,5 +77,38 @@ variable "additional_worker_pools" {
     additional_security_group_ids = optional(list(string))
   }))
   description = "List of additional worker pools with custom configurations to accommodate diverse AI workload requirements within the cluster. [Learn more](https://github.com/terraform-ibm-modules/terraform-ibm-ocp-ai/blob/main/solutions/fully-configurable/DA_docs.md#options-with-worker-pools)"
-  default     = []
+  default = [
+    {
+      pool_name         = "gpu"
+      machine_type      = "gx3.16x80.l4"
+      workers_per_zone  = 2
+      secondary_storage = "300gb.5iops-tier"
+      operating_system  = "RHEL_9_64"
```

Comment:
As discussed, default OS should be RHCOS

Reply:
Will change it but didn't changed as it is done by Arya in another PR.

Reply:
Done. Marking as resolved.